ingeniamc/ingenialink-python
# INGK-552 Create PDO script using PySOEM

### Description

Example script of EtherCAT PDOs.

Fixes #INGK-552

### Type of change

- Add script to examples
- Add complete_access argument to servo's write function.


### Tests
- Connect to an EtherCAT device and run the script. 
- Check that the motor moves and that the TPDOs registers are read.
